NEWS.rst

```diff
@@ -805,7 +805,7 @@ Bug Fixes
   with extras. (`#8785 <https://github.com/pypa/pip/issues/8785>`_)
 - New resolver: Show relevant entries from user-supplied constraint files in the
   error message to improve debuggability. (`#9300 <https://github.com/pypa/pip/issues/9300>`_)
-- Avoid parsing version to make the version check more robust against lousily
+- Avoid parsing version to make the version check more robust against loosely
```

Comment:
I'm not sure this is a typo :)

Reply:
are you sure?

```shell
❯ dict lousily
1 definition found

From The Collaborative International Dictionary of English v.0.48 [gcide]:

  Lousily \Lous"i*ly\, adv. [From {Lousy}.]
     In a lousy manner; in a mean, paltry manner; scurvily.
     [Vulgar]
     [1913 Webster]

❯ dict loosely
2 definitions found

From The Collaborative International Dictionary of English v.0.48 [gcide]:

  Loosely \Loose"ly\, adv.
     In a loose manner.
     [1913 Webster]

From WordNet (r) 3.0 (2006) [wn]:

  loosely
      adv 1: in a relaxed manner; not rigid; "his hands lay loosely"
             [syn: {loosely}, {slackly}]
      2: in a loose manner; "a union of loosely federated states"
      3: without regard to specific details or exceptions; "he
         interprets the law broadly" [syn: {broadly}, {loosely},
         {broadly speaking}, {generally}] [ant: {narrowly}]
      4: knitted in a loose manner; "loosely knit"
```

Reply:
"in a lousy manner" is the intended meaning here.

Reply:
gotcha, ok, fixed for that, force pushed